src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/histogram.js

```diff
@@ -0,0 +1,683 @@
+/** Histogram Visualization. */
+loadScript("https://d3js.org/d3.v4.min.js");
+loadStyle("https://fontlibrary.org/face/dejavu-sans-mono");
+
+let shortcuts = {
+  zoomIn: (e) => (e.ctrlKey || e.metaKey) && e.key === "z",
+  showAll: (e) => (e.ctrlKey || e.metaKey) && e.key === "a",
+};
+
+const LABEL_STYLE = "font-family: DejaVuSansMonoBook; font-size: 10px;";
+const X_AXIS_LABEL_WIDTH = 30;
+const ANIMATION_DURATION = 1000;
+const LINEAR_SCALE = "linear";
+const LIGHT_PLOT_COLOR = "#00E890";
+const DARK_PLOT_COLOR = "#E0A63B";
+const DEFAULT_NUMBER_OF_BINS = 10;
+const BUTTON_HEIGHT = 25;
```

Comment:
Some of these variables are not used anywhere. Please, check them and other functions / variables in this file. Moreover, please align all `=` here

Reply:
This doesn't seem to work with prettier. So, I'm leaving this for the moment.